Toolkit-API/Device/HardwareInfo.cs

```diff
@@ -0,0 +1,102 @@
+using System;
+using System.Collections.Generic;
+
+#if HAS_NEWTONSOFT_JSON
+using Newtonsoft.Json; // For JsonReaderException
+using Newtonsoft.Json.Linq;
+#endif
+
+namespace LookingGlass.Toolkit {
+    [Serializable]
+    public struct HardwareInfo {
+        public int index;
+        public string hardwareVersion;
+        public string hardwareVersionLong;
+        public bool hasEdidCalibration;
+        public float hfov;
+        public float vfov;
+        public float viewCone;
+        public int resolutionWidth;
+        public int resolutionHeight;
+        public string defaultQuilt;
+        public string calibration;
```

Comment:
We can rename this to like LKGDisplayInfo?

Reply:
Oh wait, so this is really what we already have, -- we could just incorporate what you created as HardwareInfo (struct) into the LKGDeviceTemplate class?

![image](https://github.com/user-attachments/assets/0f1fdfb0-c137-4a40-a5ce-7cb5bc66dec6)

I think I called them "LKG Devices" when we're talking about properties of BOTH the display and the hardware properties... but yeah, should we just replace this class, and update the implementation for LKGDeviceTemplateSystem to pull from Bridge using this new API endpoint instead of the hard-coded data in there now?

Reply:
Thanks, yeah that would be great. Feel free to let me know when this PR is ready to re-review